src/main/java/com/avispl/dal/communicator/cisco/dto/control/commands/userinterface/UserInterfaceCommand.java

```diff
@@ -0,0 +1,177 @@
+                @XmlElement(name = "Text")
+                private ValueSpaceRefHolder text;
+                @XmlElement(name = "Duration")
+                private ValueSpaceRefHolder duration;
+
+                public Display() {}
+                public Display(String text, Integer duration) {
+                    this.text = new ValueSpaceRefHolder(text);
+                    this.duration = new ValueSpaceRefHolder(duration);
+                }
+
+                /**
+                 * Retrieves {@code {@link #text}}
+                 *
+                 * @return value of {@link #text}
+                 */
+                public ValueSpaceRefHolder getText() {
+                    return text;
+                }
+
+                /**
+                 * Sets {@code text}
+                 *
+                 * @param text the {@code com.avispl.dal.communicator.cisco.dto.ValueSpaceRefHolder} field
+                 */
+                public void setText(ValueSpaceRefHolder text) {
+                    this.text = text;
+                }
+
+                /**
+                 * Retrieves {@code {@link #duration}}
+                 *
+                 * @return value of {@link #duration}
+                 */
+                public ValueSpaceRefHolder getDuration() {
+                    return duration;
+                }
+
+                /**
+                 * Sets {@code duration}
+                 *
+                 * @param duration the {@code com.avispl.dal.communicator.cisco.dto.ValueSpaceRefHolder} field
+                 */
+                public void setDuration(ValueSpaceRefHolder duration) {
+                    this.duration = duration;
+                }
+
+                /**
+                 * Retrieves {@code {@link #command}}
+                 *
+                 * @return value of {@link #command}
+                 */
+                public String getCommand() {
+                    return command;
+                }
+            }
+            @XmlElement(name = "Display")
+            private Display display;
+
+            /**
+             * Retrieves {@code {@link #display}}
+             *
+             * @return value of {@link #display}
+             */
+            public Display getDisplay() {
+                return display;
+            }
+
+            /**
+             * Sets {@code display}
+             *
+             * @param display the {@code com.avispl.dal.communicator.cisco.dto.control.commands.userinterface.UserInterfaceCommand.Message.TextLine.Display} field
+             */
+            public void setDisplay(Display display) {
+                this.display = display;
+            }
+        }
+        @XmlElement(name = "TextLine")
+        private TextLine textLine;
+
+        public Message () {}
+        public Message (String text, Integer duration) {
+            textLine = new TextLine();
+            textLine.setDisplay(new TextLine.Display(text, duration));
+        }

```

Comment:
public constructors missing javadocs.
Please check all constructors have javadocs (sorry for such tedious tasks, but if we don't do it, we'll have too many warnings)